legion/legion/k8s/utils.py

```diff
@@ -56,6 +56,60 @@
     ('ref', str)
 ])
 
+CPU_REDUCE_PAT = re.compile(r"^(\d+)(m?)$")
+MEM_REDUCE_PAT = re.compile(r"^(\d+)(E|P|T|G|M|K|Ei|Pi|Ti|Gi|Mi|Ki|)$")
+
+
+def _calculate_request(value: int) -> int:
+    """
+    Calculate request for k8s resource
+
+    :param value: k8s resource
+    :type value: int
+    :return: reduced k8s resource - int
+    """
+    return value * (100 - int(os.getenv(*legion.config.REDUCE_MODEL_REQUESTS_BY))) // 100
+
+
+def reduce_mem_resource(res: str):
+    """
+    Reduce memory k8s resource by 30%
```

Comment:
Minor comment

Maybe it is better not to hardcode the exact value in docstring? 
We can reference a variable in docs which is responsible for the percent.
And it is actually 33%, right?

Reply:
Fixed